rust/hyperlane-base/src/types/multisig.rs

```diff
@@ -344,14 +356,14 @@ impl MultisigCheckpointSyncer {
                         }
                     }
                 } else {
-                    debug!(
+                    info!(
                         validator = format!("{:#x}", validator),
                         index = index,
                         "Unable to find signed checkpoint"
                     );
                 }
             } else {
-                debug!(%validator, "Unable to find checkpoint syncer");
+                info!(%validator, "Unable to find checkpoint syncer");
```

Comment:
why change this?

Reply:
for test. just changing relayer / validator instances logger mode is make too verbose output to test. I'll recover it.